simplivity/resources/policies.py

```diff
@@ -73,6 +73,23 @@ def get_by_data(self, data):
         """
         return Policy(self._connection, self._client, data)
 
+    def create(self, name, flags=None, timeout=-1):
+        """ create the new policy
+
+        Args:
+            flags: Dictionary of filters, example: {'cluster_group_id': 'cluster_group_id'}
+            name : name of policy
+            timeout : timeout for the operation to complete
```

Comment:
Lets keep with the other phrase in the rest of the code base for timeout

i.e.

            timeout: Time out for the request in seconds.

Reply:
Updated string as per suggestion